src/pages/Home/Home.js

```diff
@@ -1,54 +1,117 @@
 import React, { Component } from "react";
 
+import { EPISODE, URL } from "../../constants/routes";
 import Layout from "../../components/Layout";
-// import EpisodeCard from "../../components/EpisodeCard";
+import EpisodeCard from "../../components/EpisodeCard";
+
+const axios = require("axios");
 
 class Home extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
-    // page: 1,
-    // paginationInfo: null,
-    // episodes: [],
-    // hasLoaded: false,
-    // hasError: false,
-    // errorMessage: null,
+    this.state = {
+      page: 1,
+      paginationInfo: null,
+      episodes: [],
+      hasLoaded: false,
+      hasError: false,
+      errorMessage: null,
+    };
+
+    this.loadEpisodes = this.loadEpisodes.bind(this);
+    this.loadMore = this.loadMore.bind(this);
   }
 
   async componentDidMount() {
-    // this.loadEpisodes();
+    const { page } = this.state;
+    await this.loadEpisodes(page);
+  }
+
+  componentDidUpdate(_prevProps, prevState) {
+    const { page: prevPage } = prevState;
+    const { page } = this.state;
+    if (prevPage !== page) {
+      this.loadEpisodes(page);
+    }
+  }
+
+  loadMore() {
+    this.setState((prevState) => ({
+      page: prevState.page + 1,
+    }));
   }
 
-  async loadEpisodes() {
-    console.log(this);
+  async loadEpisodes(page) {
+    try {
+      const response = await axios.get(`${URL}${EPISODE}?page=${page}`);
+      const data = response.data.results;
+      const info = response.data.info;
+      this.setState((prevState) => ({
+        paginationInfo: info,
+        hasLoaded: true,
+        episodes: [...prevState.episodes, ...data],
+      }));
+    } catch (error) {
+      this.setState({
+        errorMessage: error.message,
+        hasError: true,
+      });
+    }
   }
 
```

Comment:
You could use a method to change page. In my case I have this:

`loadNextPage() {
    this.setState((prevState) => {
      return { page: prevState.page + 1 };
    });
  }`

The rest of the code is very well done. It was just this missing that could improve your code.